Snippets/Unity/Extensions.DependencyInjection_1/Extensions.DependencyInjection_1.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>7.1</LangVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="NServiceBus.Extensions.DependencyInjection" Version="1.*" />
+    <PackageReference Include="Unity.Microsoft.DependencyInjection" Version="5.11.5" />
```

Comment:
are we more careful with unity dependencies in the snippets too due to our versioning policy with Unity? https://docs.particular.net/nservicebus/dependency-injection/unity#breaking-changes-in-unity

Reply:
Given the stability of the MS DI seam I think we can go with a wildcard, I will change